src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import {classNames} from '@shopify/react-utilities/styles';
+import {createUniqueIDFactory} from '@shopify/javascript-utilities/other';
+import Labelled from '../Labelled';
+
+import * as styles from './Toggle.scss';
+
+export interface Props {
+  /** Visually hidden text for screen readers */
+  accessibilityLabel?: string;
```

Comment:
Do we typically provide both an `accessibilityLabel` and a `label` prop? Wouldn't it be simpler to just use the `label` prop and hide it if `labelHidden`?

Reply:
For form components, it looks like don't provide the `accessibilityLabel`. I fixed it.